dataactbroker/handlers/fileHandler.py

```diff
@@ -1573,8 +1573,60 @@ def map_generate_status(upload_job, validation_job=None):
 
 
 def fabs_derivations(obj):
+
+    sess = GlobalDB.db().session
+
     # deriving total_funding_amount
     federal_action_obligation = obj['federal_action_obligation'] or 0
     non_federal_funding_amount = obj['non_federal_funding_amount'] or 0
     obj['total_funding_amount'] = federal_action_obligation + non_federal_funding_amount
+
+    # deriving awarding agency name
+    try:
```

Comment:
Thinking about it while not focusing on sprint review, I'm wondering if we don't want to do a try/except here. It's generally bad practice to do a default "except" statement without specifying the error to begin with, but I think it might simply be cleaner/faster to use `one_or_none()` instead of `one()`. If it doesn't find the code, it'll just return `None`. This is relevant to all cases of try/except in this PR and also the one in the other PR.